ui/StatusQ/src/StatusQ/Popups/Dialog/StatusDialog.qml

```diff
@@ -13,9 +13,9 @@ Dialog {
     property string subtitle
     /*!
        \qmlproperty destroyOnClose
-        This property decides whether the popup component should be destroyed when closed. Default value is true.
+        This property decides whether the popup component should be destroyed when closed. Default value is false.
     */
-    property bool destroyOnClose: true
+    property bool destroyOnClose: false
```

Comment:
https://github.com/status-im/status-desktop/pull/10668#discussion_r1220151161

Reply:
Understood; what should I do here? Remove the property altogether?

Reply:
I just wanted to raise awareness. I know you are not the one who introduced it, but yeah, it would be great to remove this property altogether.

Reply:
I agree 100% with @osmaczko. Components should not destroy themselves from initial implementation.

Probably what we _could_ do is
```qml
popup.closed.connect(() => { popup.destroy() })
```
somwhere in this function:
https://github.com/status-im/status-desktop/blob/e19105d8780987a27506e73a4fdaf3ddf02da0f2/ui/app/mainui/Popups.qml#L50-L68

I don't really like this solution, but at least we would only destroy popups that were created dynamically.
But IMO it was fine to destroy it right with the Component declaration.

Reply:
I agree 100% with @osmaczko. Components should not destroy themselves from initial implementation.

Probably what we _could_ do is
```qml
popup.closed.connect(() => { popup.destroy() })
```
somwhere in this function:
https://github.com/status-im/status-desktop/blob/e19105d8780987a27506e73a4fdaf3ddf02da0f2/ui/app/mainui/Popups.qml#L50-L68

I don't really like this solution, but at least we would only destroy popups that were created dynamically.
But IMO it was fine to destroy it right with the Component declaration.

Reply:
I agree 100% with @osmaczko. Components should not destroy themselves from initial implementation.

Probably what we _could_ do is
```qml
popup.closed.connect(() => { popup.destroy() })
```
somwhere in this function:
https://github.com/status-im/status-desktop/blob/e19105d8780987a27506e73a4fdaf3ddf02da0f2/ui/app/mainui/Popups.qml#L50-L68

I don't really like this solution, but at least we would only destroy popups that were created dynamically.
But IMO it was fine to destroy it right with the Component declaration.

Reply:
I see but we already do destroy the modals that live in Popups.qml :)